File: src/main/java/lk/ijse/login/entity/Citizan.java
package lk.ijse.login.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@NoArgsConstructor
@AllArgsConstructor
@Data
@Table(name = "Citizan")
public class Citizan {
        @Id
        private  int id;
        private String NIC;
        private String FirstName;
        private String LastName;
        private String ContactNumber;
        private String Address;
        private String EmailAddress;
        private String Password;

}

File: Frontend/js/citizan.js
function saveCitizan(){
    let FirstName=$(`#FirstName`).val();
    let LastName=$(`#LastName`).val();
    let NIC=$(`#Nic`).val();
    let ContactNumber=$(`#ContactNumber`).val();
    let Address=$(`#Address`).val();
    let EmailAddress=$(`#EmailAddress`).val();
    let Password=$(`#Password`).val();


    $.ajax({
        method:"POST",
        contentType:"application/json",
        url:"http://localhost:8080/api/v1/citizan/saveCitizan",
        async:true,
        data:JSON.stringify({
            "id": "",
            "address": Address,
            "contactNumber":ContactNumber,
            "emailAddress":EmailAddress,
            "firstName": FirstName,
            "lastName":LastName,
            "nic": NIC,
            "password": Password

        }),
        success:function (data){

            alert("saved")

        },
        error:function (){
            alert("Error")
        }
    })
}